test/testpaint/Compat.cpp

```diff
@@ -861,6 +861,10 @@ void ride_ratings_calculate_hybrid_coaster([[maybe_unused]] Ride* ride)
 {
 }
 
+void ride_ratings_calculate_single rail_roller_coaster([[maybe_unused]] Ride* ride)
```

Comment:
```suggestion
void ride_ratings_calculate_single_rail_roller_coaster([[maybe_unused]] Ride* ride)
```

Reply:
@Gymnasiast These two lines look identical, what needs changing exactly?

Reply:
@X123M3-256 It should be `single_rail`, not `single rail` (missing `_`)

Reply:
Oh yeah, not sure how I missed that.